terminus-ui/drawer/src/drawer.component.ts

```diff
@@ -0,0 +1,373 @@
+// tslint:disable: template-no-call-expression
+import { AnimationEvent } from '@angular/animations';
+import { hasModifierKey } from '@angular/cdk/keycodes';
+import { Platform } from '@angular/cdk/platform';
+import {
+  AfterContentChecked,
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  EventEmitter,
+  HostListener,
+  Input,
+  NgZone,
+  OnDestroy,
+  Output,
+  Renderer2,
+  ViewEncapsulation,
+} from '@angular/core';
+import {
+  isUnset,
+  KEYS,
+  untilComponentDestroyed,
+} from '@terminus/ngx-tools';
+import {
+  fromEvent,
+  Observable,
+  Subject,
+} from 'rxjs';
+import {
+  distinctUntilChanged,
+  filter,
+  map,
+  take,
+} from 'rxjs/operators';
+
+import { tsDrawerAnimations } from './drawer-animations';
+
+/**
+ * Result of the toggle promise that indicates the state of the drawer.
+ */
```

Comment:
I was under the false impression that if the comment is longer than one line we use `/** */` style and if it's within one line we use `//` instead.... guess it's not.
It's all updated now.